### PR TITLE
Containers cleanups for v5

### LINKED ISSUE #2664: PgBackrest latest version - packaged container
Hi,

Currently the last package version that I found for the docker image crunchy-pgBackrest is v2.33.-2.
Is there a plan to get the v2.34 or the v2.35 of pgbackrest in crunchy docker image?

Thank you for your reply and for your work.

ps: if it's not the right place for this question, let me know.

### PR DESCRIPTION
**Pull request by jkatz**
This contains a series of cleanups for PGO v5.

Issue: [sc-11951]
Issue: [sc-12609]
Issue: [sc-12610]
closes CrunchyData/postgres-operator#2664

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,52 +40,39 @@ ifeq ("$(IMGBUILDER)","docker")
...
 endif
 
 .PHONY:	all license pgbackrest-images pg-independent-images pgimages
 
 # list of image names, helpful in pushing
 images = crunchy-postgres \
-	crunchy-postgres-ha \
 	crunchy-upgrade \
 	crunchy-pgbackrest \
 	crunchy-pgbackrest-repo \
```
Comment (andrewlecuyer) [inline Makefile]: Just noting that `crunchy-pgbackrest-repo` is no longer used with PGO v5, and can therefore be removed.

**Comment by crunchyheath** [comment]
For pull-from-crunchy.sh, pull-from-dockerhub.sh, push-to-local.sh and remove-all-images.sh should you also remove the references to pgpool, pgbadger, and pgadmin4 since these containers were also removed from the Makefile target list?  Other than that comment everything else looks okay to me.

**Review by andrewlecuyer — APPROVED** [review]

**Comment by andrewlecuyer** [comment]
Noting that I tested a build of all containers with PGO v5, and did not run into any issues.

**Comment by jkatz** [comment]
All feedback addressed.

**Review by andrewlecuyer — APPROVED** [review]

File: docs/content/overview/overview.md [hunk 2]
```diff
@@ -11,16 +11,12 @@ The following provides a high level overview of each of the container images.
 
 ## CentOS vs Red Hat UBI Images
 
-The Crunchy Container suite provides four different OS images: `centos7`, `centos8`, `ubi7`, and `ubi8-minimal`.  Both images
+The Crunchy Container suite provides four different OS images: `centos8` and `ubi8-minimal`.  Both images
```
Comment (cbandy) [inline docs/content/overview/overview.md]:
```suggestion
The Crunchy Container suite provides two different OS images: `centos8` and `ubi8-minimal`.  Both images
```

**Pull request merged**